src/components/KeyblockDetailsPanel.vue

```diff
@@ -293,25 +279,19 @@ const isNextKeyblockMined = computed(() =>
     align-items: center;
   }
 
-  &__button {
-    width: 32px;
-    height: 32px;
+  &__next-button {
```

Comment:
It's not necessarily bad, but what do you think about writing it with alignment with the child component 
```suggestion
  &__button--next {
```